core/includes/config.inc

```diff
@@ -685,6 +715,8 @@ class Config {
  */
 interface ConfigStorageInterface {
 
+  public function initializeStorage();
```

Comment:
Yup.  And unit tests specifically for all of the `ConfigStorageInterface` classes.